### ISSUE TITLE
Creator's Update for Windows 10 broke Audioswitch 2.2.1.0. (Oct/2017)

### ISSUE THREAD

**Issue by RobinBrotherston** [sirWest/AudioSwitch#134]
Doesn't seem to load at all.

**Comment by frogwallet** [comment]
Having the same issue. Have tried reinstalling the software (after complete removal) which didn't work.

Changing the executable to run in compatibility mode for windows 8 did however.

**Comment by ocombe** [comment]
You don't really need this app with creators update, you can now quick switch with the default sound icon: click on it, click on the small arrow at the top right and it opens the quick switch:

![quick-switch](https://user-images.githubusercontent.com/265378/31888262-9c346cb2-b7fb-11e7-8e8e-e9fbd4c1fcc9.jpg)

![quick-switch2](https://user-images.githubusercontent.com/265378/31888264-9e658da4-b7fb-11e7-9b22-5c470fc166f4.jpg)

**Comment by RobinBrotherston** [comment]
Yes, but AudioSwitch you could attach a keybind to it. Handy when you are
in a application.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by wojtek-abakus** [comment]
It can be easily fixed... just change line 207 in Program.cs from:
if (mutex.WaitOne(0, false))
to:
if (mutex.WaitOne(10, false))

I have no idea why the mutex behavior has changed but that's the reason

**Comment by beppe9000** [comment]
I use the latest release of AudioSwitch on a fresh install of windows 10 without any kind of problem. I wonder why is this issue still open... i was going to be sad for not being able to use it after updating if I had not tried.

<hr>

> You don't really need this app with creators update, you can now quick switch with the default sound icon: click on it, click on the small arrow at the top right and it opens the quick switch.

By the way you didn't really need the creators update to do that.
